### DISCUSSION TITLE
iOS APP error: Invalid argument(s): Invalid locale "nap_IT"

### DISCUSSION THREAD

**Discussion by spad-0x** [immich-app/immich#9468] [Q&A]
When open the app also at first startup I have this issue, I tried to open the app with same account on another iPhone and it's all ok. Only on my iPhone I have the error

![IMG_B9915BF76A74-1](https://github.com/immich-app/immich/assets/43964956/07d800b8-1d54-4a83-b1c8-731b1c350730)
[Immich_log_2024-05-14T12:53:02.236723.log](https://github.com/immich-app/immich/files/15307505/Immich_log_2024-05-14T12.53.02.236723.log)

**Comment by italianog** [comment]
Hi, we had the same problem. Your iPhone locale is set to "Napoletano" (Italian dialect) and you don't support it

**Comment by metri** [comment]
Hi, I had the same problem with "cv_RU" (the Chuvash language with cyrillic).

**Reply by metri** [reply]
I switched to the English localization to not have a problem.  :/
